velox/core/QueryConfig.h

```diff
@@ -689,6 +697,16 @@ class QueryConfig {
     return get<std::string>(kQueryTraceNodeIds, "");
   }
 
+  uint64_t queryTraceMaxBytes() const {
+    // The default query trace bytes, 0 by default.
```

Comment:
This comment is redundant. It simply repeats the code on the next line. Let's remove.